numba/tests/test_parfors.py

```diff
@@ -1018,6 +1018,24 @@ def test_impl(n):
         self.assertEqual(parfor_output, py_output)
         self.assertEqual(countParfors(test_impl, (types.int64, )), 0)
 
+    def test_stararg(self):
+        def test_impl(n):
+            A = np.random.randn(*n)
```

Comment:
```suggestion
            np.random.seed(0)
            A = np.random.randn(*n)
```
suggest setting the seed for reproducible testing.